unlock-protocol-com/blog/gitcoin-round-15.md

```diff
@@ -0,0 +1,108 @@
+around use of Active Blockreference. 
+
+## web3 Social
+
+We saw an increase in the number of web3 social projects taking advantage of 
+the power of Unlock Protocol too! [Nimi](https://nimi.io/) was a crowd favorite 
+with an dApp that allows you to aggregate your soical profiles. Other notable 
+projects include [Cobogo](https://cobogo.io) for creator profiles and 
+fundraising for the creator economy, [Maven Foundation](https://maven.foundation/) 
+is continuing their work on building tools to empower social influencers and 
+[MeTag](https://www.getmetag.io) is creating a scannable QR tag which is also NFC 
+enabled to help users share their web3 wallet information and social profiles. 
+[Headline](https://viaheadline.xyz/) a decentrailized publishing platform which 
+got it's start with a [UDT Grant](https://docs.unlock-protocol.com/governance/grants-bounties-and-matchings) from Unlock Protocol jumped onto the Gitcoin 
+platform this round to continue their work.
+
+## Education & Security
+
+[VestLab](VestLab.io) is aiming to get you all the information about a crypto 
+project before you invest. While [White Hat DAO](https://whitehatdao.com) is 
+looking to give deeper insights on blockchain projects with offerings that 
+include smart contract audits, safety ratings along with tokenomics analysis and 
+due-diligence. [Fractionalize](https://www.amwwaldorf.org) is implementing a 
+first-of-its-kind public education program to teach kids about web3 and NFTs and 
+the [Decentralized Library](https://decentralized-library.vercel.app/) will 
+include Unlock NFT library cards along with reading rewards stored in Unlock 
+NFTs for "proof of knowledge". 
+
+## Unicorns
+
+We also some projects so unique we're daring to call theme unicorns. 
+[Trust Finance](https://www.trustfin.xyz) is looking to solve some challenges in 
+the traditional payroll and contractor payments space like slow posting times 
+and by making it easier for workers to invest a portion of their paychecks. 
+[Starksheet](https://www.starksheet.xyz/) is hoping to tackle some the largest 
+data silos by decentralizing business intelligence with a web3 spreadsheet 
+solution making use of Unlock NFTs.
+
+## Diversity, Equity & Inclusion
+
+Last but not least there were several projects both utilizing Unlock Protocol 
+and tackling Diversity, Equity & Inclusiion (DEI) in web3. We continue to be 
+passionate about our support of these projects and were so enthusiastic to see 
+so many in this round. Our friends over at [Blu3 DAO](https://www.blu3dao.com/) 
+continue to do such impressive work educating, preparing and sponsoring hackers 
+in the web3 space. The community seems to agree with our take on the work they 
+do with hundreds of supporters and thousands of dollars in community support they 
+were able to raise close to $60k this round with matching funds and we couldn't 
+be happier to see that happen for them! 
```

Comment:
"The community seems to agree with our take on the work they 
do with hundreds of supporters and thousands of dollars in community support they were able to raise close to $60k this round with matching funds and we couldn't be happier to see that happen for them!" is a run-on. Please change to:

The community seems to agree with our take on the work they 
do, with hundreds of supporters and thousands of dollars in community support. They were able to raise close to $60k this round with matching funds and we couldn't be happier to see that happen for them!